apps/codeforafrica/src/components/NavList/index.js

```diff
@@ -0,0 +1,48 @@
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavUl = styled("ul", {
+  shouldForwardProp: (prop) => prop !== "footer",
+  slot: "Root",
+  overridesResolver: (props, styles) => {
+    const { footer } = props.ownerState;
+    return [styles.root, footer && styles.footer];
+  },
+})(({ theme: { breakpoints }, ownerState }) => ({
+  ...(!ownerState?.footer && {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "flex-start",
+    alignItem: "flex-start",
+    [breakpoints.up("lg")]: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "flex-end",
+      alignItem: "center",
+    },
+  }),
+}));
+
+const NavList = React.forwardRef(function NavList(props, ref) {
+  const { footer = true, children, ...others } = props;
```

Comment:
```suggestion
  const { direction = "column", children, ...others } = props;
```

I think calling it `direction` is better because the `NavList` looks the same on Footer as well as Mobile navigation.